src/components/InputField/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { KeyboardEvent } from "react";
+import './index.css';
+
+interface IInputFieldProps {
+  visible: boolean,
+  onEnterClick: (value: string) => void,
+  onToggleAllClick: () => void
+}
+
+const InputField: React.FC<IInputFieldProps> = ({ visible, onEnterClick, onToggleAllClick }) => {
+  const handleKeyDown = (event: KeyboardEvent<HTMLInputElement>) => {
+    if(event.key !== 'Enter' || (event.target as HTMLInputElement).value === '') {
```

Comment:
For long conditions it is best to create separate constants
For example 
const enterKey = event.key === 'Enter'